packages/components/avatar/src/index.ts

```diff
@@ -1,6 +1,3 @@
-import type { TAvatarProps as AvatarProps } from './avatar';
-
 export { default } from './avatar';
 export { default as version } from './version';
-
-export type TAvatarProps = AvatarProps;
+export * from './export-types';
```

Comment:
When preconstruct builds the dist files for "development", it uses a symlink to the package entry point `src/index.ts`. However, the file is a JS file, not a TS file, so any TS syntax is invalid.

To fix that, we define the type exports in a separate file and simply re-export everything from that file from the entry point.